cinder/volume/drivers/vmware/datastore.py

```diff
@@ -226,6 +230,7 @@ def _get_object_properties(self, obj_content):
                 props = {prop.name: prop.val for prop in prop_set}
         return props
 
+    @cached(cache=TTLCache(maxsize=128, ttl=30))
```

Comment:
We fetch the `summary` here. If that's used to compute free space somewhere, we will risk overprovisioning.